expressjs/src/routes/index.js

```diff
@@ -24,6 +27,14 @@ module.exports = app => {
   })
 }
 
+function shouldRenderJson(req) {
+  const ua = req.header('user-agent')
+  if (ua && ua.startsWith('Mozilla')) {
```

Comment:
Are you sure that all browser's agent start with "Mozilla" ? I don't think so, I think we should be more selective and check that at least for Safari, Chrome, Firefox, MS Edge you get back a web page. Or there is even a better way to detect a browser.

Reply:
All modern browsers have that. I did a check. See: https://www.whatismybrowser.com/guides/the-latest-user-agent/